content/en/blog/2025/expose-otel-collector-gateway-api/index.md

```diff
@@ -0,0 +1,621 @@
+at the time of writing, if you are using Istio, ensure that
+`PILOT_ENABLE_ALPHA_GATEWAY_API` is enabled during the install.
+
+{{% /alert %}}
+
+## What is the Kubernetes Gateway API?
+
+The [Kubernetes Gateway API](https://gateway-api.sigs.k8s.io/) is an evolution
+of the older Ingress API. It provides a more expressive, role-oriented, and
+flexible way to manage inbound traffic to your cluster. The
+[GAMMA initiative](https://gateway-api.sigs.k8s.io/mesh/gamma/) defines the
+Gateway API implementation. It was introduced for the following reasons:
+
+- **Limitations of Ingress:** The Ingress API, while useful, became limiting. It
+  lacked standardization across implementations and also limited routing
+  capabilities that varied widely across different implementations.
+- **Role Separation:** Gateway API separates concerns:
+  - `GatewayClass`: Defines a _type_ of load balancer (e.g., Istio, GKE LB).
+    Managed by Infrastructure Admins.
+  - `Gateway`: Represents an instance of a load balancer requesting a specific
+    `GatewayClass`. Defines listeners (ports, protocols, TLS). Managed by
+    Cluster Operators. They can be shared across namespaces as well.
+  - `HTTPRoute`, `GRPCRoute`, `TCPRoute`, `TLSRoute`, etc.: Define
+    application-level routing rules, attaching to a `Gateway`. Managed by
+    Application Developers/Owners.
+- **Portability:** Standardized API definition aims for greater portability
+  across different underlying gateway/service mesh implementations.
+- **Expressiveness:** Natively supports advanced features like header
+  manipulation, traffic splitting, mTLS configuration, gRPC routing, and more.
+
+In essence, Gateway API offers a more robust and standardized model for managing
+north-south traffic compared to the traditional Ingress API.
+
+## mTLS - A brief introduction
+
+Mutual TLS (mTLS) extends standard TLS by requiring _both_ the client and server
+to present and validate certificates for mutual authentication.
+
+Standard TLS (like HTTPS on websites) verifies the _server's_ identity to the
+_client_. Mutual TLS (mTLS) goes a step further:
+
+- The client verifies the server's identity (using the server's certificate).
+- The server _also_ verifies the client's identity (using the client's
+  certificate).
+
+Mutual TLS is important because it provides strong authentication, ensures
+end-to-end encryption, and aligns with Zero Trust security principles:
+
+- Ensures _only_ trusted clients (those possessing a valid certificate signed by
+  trusted CA) can connect to your exposed service.
+
+- All communication between the authenticated client and the server (such as a
+  Gateway) is encrypted.
+
+- It supports a Zero Trust model by requiring verification from both parties,
+  never assuming trust by default.
+
+## The Scenario
+
+Here are the steps we'll follow to expose an OTel Collector deployment outside
+the cluster.
+
+1.  Deploy an OTel Collector inside Kubernetes configured with a simple
```

Comment:
This is a bit of a nit, but the numbering of these steps confused me because they don't line up with the "Step N" headings that follow. I would suggest changing this to an unordered list or making the ordered list match the step headings.